codechain/keystore/CCkey.py

```diff
@@ -0,0 +1,44 @@
+from jsondb.db import Database
+
+from .KeystoreManager import keystoreManager
+from .KeyType import KeyType
+
+
+class CCkey:
+    def __init__(self, db_path):
+        self.platform = keystoreManager(KeyType.PLATFORM, db_path)
+        self.asset = keystoreManager(KeyType.ASSET, db_path)
+        self.db_path = db_path
+
+    @staticmethod
+    def create(**kwargs):
+        db_path = kwargs.get("db_path", "keystore.db")
+
+        return CCkey(db_path)
+
+    def exist(self, **kwargs):
+        db_path = kwargs.get("db_path", "keystore.db")
+        db = Database(db_path)
+
+        meta = db["meta"]
+        platform = db["platform"]
+        asset = db["asset"]
+        hdwseed = db["hdwseed"]
+
+        return (
+            (meta is not None and meta != "")
+            or (platform is not None and len(platform) != 0)
+            or (asset is not None and len(asset) != 0)
+            or (hdwseed is not None and len(hdwseed) != 0)
+        )
+
+    def get_meta(self):
+        db = Database(self.db_path)

```

Comment:
You can remove this `db = Database(self.db_path)` with `self.db` if you change the design.

Reply:
I replaced it.